itests/path_type_filters_test.go

```diff
@@ -1,200 +0,0 @@
-package itests
-
-import (
-	"context"
-	"strings"
-	"testing"
-	"time"
-
-	logging "github.com/ipfs/go-log/v2"
-	"github.com/stretchr/testify/require"
-
-	"github.com/filecoin-project/lotus/itests/kit"
-	"github.com/filecoin-project/lotus/storage/sealer/sealtasks"
-	"github.com/filecoin-project/lotus/storage/sealer/storiface"
-)
-
-func TestPathTypeFilters(t *testing.T) {
-	kit.QuietMiningLogs()
-
-	runTest := func(t *testing.T, name string, asserts func(t *testing.T, ctx context.Context, miner *kit.TestMiner, run func())) {
-		t.Run(name, func(t *testing.T) {
-			ctx, cancel := context.WithCancel(context.Background())
-			defer cancel()
-
-			_ = logging.SetLogLevel("storageminer", "INFO")
-
-			var (
-				client   kit.TestFullNode
-				miner    kit.TestMiner
-				wiw, wdw kit.TestWorker
-			)
-			ens := kit.NewEnsemble(t, kit.LatestActorsAt(-1)).
-				FullNode(&client, kit.ThroughRPC()).
-				Miner(&miner, &client, kit.WithAllSubsystems(), kit.ThroughRPC(), kit.PresealSectors(2), kit.NoStorage()).
-				Worker(&miner, &wiw, kit.ThroughRPC(), kit.NoStorage(), kit.WithTaskTypes([]sealtasks.TaskType{sealtasks.TTGenerateWinningPoSt})).
-				Worker(&miner, &wdw, kit.ThroughRPC(), kit.NoStorage(), kit.WithTaskTypes([]sealtasks.TaskType{sealtasks.TTGenerateWindowPoSt})).
-				Start()
-
-			ens.InterconnectAll().BeginMiningMustPost(2 * time.Millisecond)
-
-			asserts(t, ctx, &miner, func() {
-				dh := kit.NewDealHarness(t, &client, &miner, &miner)
-				dh.RunConcurrentDeals(kit.RunConcurrentDealsOpts{N: 1})
```

Comment:
oh, this test seems like a bit of a loss all because it needs deals to exercise storage; it'd be good to get some curio ppl feedback on this one

Reply:
I think this should get covered by Curio tests. I am not sure if it is great ROI building a lot of testing deal onboarding infra just for this.